scripts/install_bitcoind.sh

```diff
@@ -2,9 +2,9 @@
 
 set -ev
 
-BITCOIND_VERSION=${BITCOIN_VERSION:-25.0}
+BITCOIND_VERSION=${BITCOIN_VERSION:-25}
```

Comment:
I think we should use one place to manage the version, to avoid mistakes such as 994a1426d6

Reply:
Maybe this is a dumb question but why isn't this number 26?

Reply:
This is just the fallback version in case no environment variable is set. We set the version we actually want here: https://github.com/lightningnetwork/lnd/blob/ff3e8da1756572b0dd979a39b139a969498772dd/.github/workflows/main.yml#L24

But I guess we shouldn't do that to avoid confusion...